### DISCUSSION TITLE
D/K/G CSV Export

### DISCUSSION THREAD

**Discussion by Arwidosos** [MKT-Playbook/Hub#2] [Feature requests]
I love the way MKT-Playbook works, and it is easy to keep the Data Up to date.
Sometimes I want to use one of the other MKT Tools like DKR.exe or a Spreadsheet.
It would be a great benefit, if I could export my Data from MKT-Playbook for use in other Tools.

**Comment by TitusPosiadly** [comment]
Hey! Yes, great idea. I wanted to do something like this before but I wasn't sure what the most wanted resources were. Could you link me to the spreadsheets you'd be interested in, so I know what format the export needs to have? Or if you have a specific (universal) format already in mind let me know what that would look like. Thanks for the suggestion ✌️

EDIT:
The CSV Exporter is now part of the app! Implemented in version 1.2.0 -> #5

**Reply by Arwidosos** [reply]
As Spreadsheet examples you could use: 

https://docs.google.com/spreadsheets/d/1sSKPB3x0_n92RisYjw5pxD6_voFAujY7mq_ireQ9grM/copy

or

http://mkt.itsupthere.space/

but in General, the most Sheets works with the same input data:

Column 1 - D/K/G Name - (Dry Bowser) --> Game Name of Driver, Kart or Glider
Column 2 - Level - (6) --> Level of D/K/G
Column 3 - Base Points - (980) --> Base point of D/K/G

Advanced 

Column 4 - Level Ticket (2) --> Used Level Tickets for next Level


Example CSV Simple:
```
"Baby Mario",7,720
"Baby Peach",6,600
"Baby Daisy",5,616
"Baby Rosalina",3,560
```

Example CSV Avanced:
```
"Baby Mario",7,720,10
"Baby Peach",6,600,9
"Baby Daisy",5,616,3
"Baby Rosalina",3,560,14
```

Developer hint:
Make the delimiter configurable between "," and ";" so there is no hazzle for users using a diffrent language in excel

Example CSV Avanced for non English Excel Users:
```
"Baby Mario";7;720;10
"Baby Peach";6;600;9
"Baby Daisy",5,616,3
"Baby Rosalina";3;560;14
```

**Reply by TitusPosiadly** [reply]
Alright, thank you so much for putting this together. I will try to implement this for v1.2.0 coming after the big MKT update.
